app/services/swagger-service.js

```diff
@@ -1,21 +1,22 @@
 /*global $*/
 import Ember from 'ember';
+import { purecloudEnvironmentTld } from '../helpers/purecloud-environment';
 
 export default Ember.Service.extend({
 	swagger: {},
 	init() {
 		let that = this;
-		let swaggerUrl = '/swagger-schema/publicapi-v2-latest.json';
+		let purecloudEnvironment = purecloudEnvironmentTld();
+        let swaggerUrl = `https://api.${purecloudEnvironment}/api/v2/docs/swagger`;
 
-		if (window.location.host.indexOf('localhost') > -1) {
-			swaggerUrl = '/publicapi-v2-latest.json';
-			console.warn("Using local publicapi-v2-latest.json file, this may be outdated!");
-		}
+        const withCredentials = window.location.host.indexOf('localhost') > -1 
+			? false
+			: true;
 
 		$.getJSON({
 			url: swaggerUrl,
 			xhrFields: {
-				withCredentials: true
+				withCredentials
```

Comment:
When running on localhost and withCredentials set to true, I was getting a CORS error in the console.  I set localhost to use false to work around that error.

Reply:
The `withCredentials` setting can be removed entirely. It's only relevant when using cookies or other auth types, which the swagger resource does not use nor allow. https://developer.mozilla.org/en-US/docs/Web/API/XMLHttpRequest/withCredentials

Reply:
I removed them.  It's pulling in the metrics and dimensions from the swagger defs now.